packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -115,6 +115,27 @@
   </screenshots>
 
   <releases>
+   <release version="2024.08.15-21" date="2024-08-15">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>OpenStreetMap data as of August 10</li>
+           <li>Improved routing in cases of bad surface and missing speed limit</li>
+           <li>Display road shields in India</li>
+           <li>Self-service and outdoor seating support</li>
+           <li>Use NotoSans font instead of Serif for Hebrew</li>
+         </ul>
+       <p>Linux Desktop:</p>
+         <ul>
+           <li>Add transliteration enable/disable setting</li>
+         </ul>
```

Comment:
This can be an item in the above list. There's no sense in making a different Desktop clause here, as all linux builds are "desktop".

Reply:
Done